modules/intro/pages/multigraph-overview.adoc

```diff
@@ -8,10 +8,10 @@ MultiGraph enables several powerful use cases:
 
 * Multiple Tenancy: Use one TigerGraph instance to support several completely separate data sets, each with its own set of users. Each user community cannot see the other user communities or other data sets.
 * Fine-grained privileges on the same set of data: Role-based access control, available on single graphs, grants permission for the privilege to run queries (include data modification queries). In a single graph scheme, there is no way to say "Query X can be run by some users but not by others." Using multiple graphs defined over the same set of data, each graph can have its own set of queries and own set of users, in effect customizing who can run which queries.
-* Overlapping graphs: Graphs can partially overlap, to enable a combination of shared and private data.
-* Hierarchical subgraphs: A Graph X can be defined to cover the domains of Graphs Y and Z, that is, Graph X = (Graph Y) U (Graph Z). This provides an interesting way to describe a data partitioning or parent-child structure. (This is not the same as defining sub-classes of data types; data types are still independent.)
+* Overlapping graphs: Graphs can partially overlap to enable a combination of shared and private data.
+* Hierarchical subgraphs: A graph X can be defined to cover the domains of Graphs Y and Z, that is, Graph X = (Graph Y) U (Graph Z). This provides an interesting way to describe a data partitioning or parent-child structure. (This is not the same as defining sub-classes of data types; data types are still independent.)
```

Comment:
Inconsistency: If `graph` is capitalized in `Graph Y`, then `Graph X` should be capitalized.
Note that in my equation (I think I am the original author), the variables are `Graph X`, `Graph Y`, and `Graph Z`, not `X`, `Y`, and `Z`. You could rewrite it.